pkg/cmd/admin/policy/cani.go

```diff
@@ -125,6 +128,17 @@ func (o *canIOptions) Complete(f *clientcmd.Factory, args []string) error {
 	o.RulesReviewClient = oclient
 	o.SARClient = oclient
 
+	output := kcmdutil.GetFlagString(cmd, "output")
+	if len(output) > 0 {
+		if err != nil {
+			return err
+		}
+
+	}
+
+	printer, err := f.PrinterForCommand(cmd, false, nil, printers.PrintOptions{})
```

Comment:
@fabianofranz I have not yet tested flags like `--sort-by` with this PR; 
@abstractj maybe we should handle options like `no-headers` here:
```go
printers.PrintOptions{
    NoHeaders: kcmdutil.GetFlagBool(cmd, "no-headers"),
}
```

Also, `-o name` renders the output: `<unknown>/<unknown>`, although not sure if that might be related to [this pr](https://github.com/openshift/origin/issues/15020)